react-forms/react-forms/src/RegistrationFormControlled.tsx

```diff
@@ -0,0 +1,25 @@
+import { FormEvent, useState } from 'react';
+
+export function RegistrationFormControlled() {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  function handleSubmit(event: FormEvent) {
+    event.preventDefault();
+    console.log(`{ ${username}, ${password} }`);
```

Comment:
You will get the same output (and probably more useful) with just `console.log({ username, password });`